<?xml version="1.0" encoding="UTF-8" ?>
<testsuites>
  <testsuite errors="0" failures="1" hostname="Nicolass-iMac.local" id="0" name="FooterTagLibTests" package="racetrack" tests="1" time="0.926" timestamp="2012-03-07T22:02:58">
      <properties />

      <testcase classname="racetrack.FooterTagLibTests" name="testSomething" time="0.754">
          <failure message="java.lang.AssertionError: Implement me" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: java.lang.AssertionError: Implement me
	at racetrack.FooterTagLibTests.testSomething(FooterTagLibTests.groovy:15)
</failure>

      </testcase>

      <system-out><![CDATA[--Output from testSomething--
]]></system-out>

      <system-err><![CDATA[--Output from testSomething--
]]></system-err>

  </testsuite>
  <testsuite errors="0" failures="0" hostname="Nicolass-iMac.local" id="1" name="RaceTests" package="racetrack" tests="1" time="0.469" timestamp="2012-03-07T22:02:59">
      <properties />

      <testcase classname="racetrack.RaceTests" name="testInMiles" time="0.379" />

      <system-out><![CDATA[--Output from testInMiles--
]]></system-out>

      <system-err><![CDATA[--Output from testInMiles--
]]></system-err>

  </testsuite>
  <testsuite errors="0" failures="5" hostname="Nicolass-iMac.local" id="2" name="UserControllerTests" package="racetrack" tests="8" time="1.396" timestamp="2012-03-07T22:03:00">
      <properties />

      <testcase classname="racetrack.UserControllerTests" name="testIndex" time="0.37" />

      <testcase classname="racetrack.UserControllerTests" name="testList" time="0.364" />

      <testcase classname="racetrack.UserControllerTests" name="testCreate" time="0.085" />

      <testcase classname="racetrack.UserControllerTests" name="testSave" time="0.15">
          <failure message="Assertion failed: &#xa;&#xa;assert response.redirectedUrl == &apos;/user/show/1&apos;&#xa;       |        |             |&#xa;       |        null          false&#xa;       org.codehaus.groovy.grails.plugins.testing.GrailsMockHttpServletResponse@db5eaed&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert response.redirectedUrl == &apos;/user/show/1&apos;
       |        |             |
       |        null          false
       org.codehaus.groovy.grails.plugins.testing.GrailsMockHttpServletResponse@db5eaed

	at racetrack.UserControllerTests.testSave(UserControllerTests.groovy:49)
</failure>

      </testcase>

      <testcase classname="racetrack.UserControllerTests" name="testShow" time="0.174">
          <failure message="Assertion failed: &#xa;&#xa;assert user.save() != null&#xa;       |    |      |&#xa;       |    null   false&#xa;       racetrack.User@33409c28 (toString() == null)&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert user.save() != null
       |    |      |
       |    null   false
       racetrack.User@33409c28 (toString() == null)

	at racetrack.UserControllerTests.testShow(UserControllerTests.groovy:64)
</failure>

      </testcase>

      <testcase classname="racetrack.UserControllerTests" name="testEdit" time="0.069">
          <failure message="Assertion failed: &#xa;&#xa;assert user.save() != null&#xa;       |    |      |&#xa;       |    null   false&#xa;       racetrack.User@742cb491 (toString() == null)&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert user.save() != null
       |    |      |
       |    null   false
       racetrack.User@742cb491 (toString() == null)

	at racetrack.UserControllerTests.testEdit(UserControllerTests.groovy:83)
</failure>

      </testcase>

      <testcase classname="racetrack.UserControllerTests" name="testUpdate" time="0.063">
          <failure message="Assertion failed: &#xa;&#xa;assert user.save() != null&#xa;       |    |      |&#xa;       |    null   false&#xa;       racetrack.User@ef1a78 (toString() == null)&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert user.save() != null
       |    |      |
       |    null   false
       racetrack.User@ef1a78 (toString() == null)

	at racetrack.UserControllerTests.testUpdate(UserControllerTests.groovy:104)
</failure>

      </testcase>

      <testcase classname="racetrack.UserControllerTests" name="testDelete" time="0.061">
          <failure message="Assertion failed: &#xa;&#xa;assert user.save() != null&#xa;       |    |      |&#xa;       |    null   false&#xa;       racetrack.User@58aa23e5 (toString() == null)&#xa;" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Assertion failed: 

assert user.save() != null
       |    |      |
       |    null   false
       racetrack.User@58aa23e5 (toString() == null)

	at racetrack.UserControllerTests.testDelete(UserControllerTests.groovy:148)
</failure>

      </testcase>

      <system-out><![CDATA[--Output from testIndex--
--Output from testList--
--Output from testCreate--
--Output from testSave--
--Output from testShow--
--Output from testEdit--
--Output from testUpdate--
--Output from testDelete--
]]></system-out>

      <system-err><![CDATA[--Output from testIndex--
--Output from testList--
--Output from testCreate--
--Output from testSave--
--Output from testShow--
--Output from testEdit--
--Output from testUpdate--
--Output from testDelete--
]]></system-err>

  </testsuite>
  <testsuite errors="0" failures="1" hostname="Nicolass-iMac.local" id="3" name="UserTests" package="racetrack" tests="1" time="0.047" timestamp="2012-03-07T22:03:01">
      <properties />

      <testcase classname="racetrack.UserTests" name="testSomething" time="0.045">
          <failure message="java.lang.AssertionError: Implement me" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: java.lang.AssertionError: Implement me
	at racetrack.UserTests.testSomething(UserTests.groovy:15)
</failure>

      </testcase>

      <system-out><![CDATA[--Output from testSomething--
]]></system-out>

      <system-err><![CDATA[--Output from testSomething--
]]></system-err>

  </testsuite>
</testsuites>
